charts/query-exporter/values.yaml

```diff
@@ -0,0 +1,155 @@
+---
+replicaCount: 1
+
+image:
+  repository: adonato/query-exporter
+  tag: 2.7.0
+  pullPolicy: IfNotPresent
+  command: ["query-exporter", "-H", "0.0.0.0", "--", "/config/config.yaml"]
+
+nameOverride: "query-exporter"
+fullnameOverride: "query-exporter"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+#securityContext:
+#  allowPrivilegeEscalation: false
+#  runAsUser: 999
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 9560
+  targetPort: 9560
+
+resources:
+  limits:
+#    cpu: 50m -- Don't recommend using cpu limits. It can shot in your leg.
+    memory: 512Mi
+  requests:
+    cpu: 10m
+    memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+strategy:
+  rollingUpdate:
+    maxSurge: 1
+    maxUnavailable: 0
+  type: RollingUpdate
+
+env: {}
+#  FOO: bar
+
+# For db_connetrions_string please use this template
+# dialect[+driver]://[username:password][@host:port]/database[?option=value&...]
+
+# For db_connections ssl string use this template
+#  DB_CONNECTION_STRING : mysql+mysqldb://user:password@server.com/db?ssl_ca=/etc/ssl/certs/query-exporter-cert-db.pem
+
+secrets: {}
+#  DB_CONNECTION_STRING: postgresql://postgres:password@pghost:5432/postgres
+
+# existingSecret uses a secret name in Kubernetes
+# if existingSecret described Kubernetes secret will not be created
+
+# Highly recommend using existing existingSecret
+### Example
+# kubectl create secret generic query-exporter -n monitoring
+# kubectl patch secret query-exporter -n monitoring -p='{"data":{"db_dsn": "<db_dsn_in_base64"}}' -v=1
+# existingSecret: query-exporter
+###
+existingSecret: ""
+
+readinessProbe:
+  failureThreshold: 4
+  httpGet:
+    path: /
+    port: 9560
+    scheme: HTTP
+  initialDelaySeconds: 60
+  periodSeconds: 30
+  successThreshold: 1
+  timeoutSeconds: 2
+
+livenessProbe:
+  failureThreshold: 2
+  httpGet:
+    path: /
+    port: 9560
+    scheme: HTTP
+  initialDelaySeconds: 60
+  periodSeconds: 30
+  successThreshold: 1
+  timeoutSeconds: 2
+
+extraVolumes: {}
+#  - name: query-exporter-conf
+#    configMap:
+#      name: query-exporter-queries
+#      items:
+#        - key: config.yaml
+#          path: config.yaml
+
+extraVolumeMounts: {}
+#  - mountPath: /config
+#    readOnly: true
+#    name: query-exporter-conf
+
+# SSL Implementation
+#extraVolumeMounts: 
+#- mountPath: /etc/ssl/certs/query-exporter-cert-db.pem
```

Comment:
Were there AWS docs that you were referencing?  

Where can you get this cert from?

etc

Reply:
you said: Were there AWS docs that you were referencing?
you said: Where can you get this cert from?
answer: https://github.com/exact-payments/gruntwork-infrastructure-live/pull/1170

----

I will not answer this question here , because here it is something generic ; this certificate can be obtained from wherever you want; it is a simple text; the bds can't exactly be RDS, there can be a mysql bare metal installation and create your own certificate; at this point everything is generic;

I will answer the specific questions in the pr that will consume this.

Reply:
Yeah..the problem with being specific and generic is a fine line.  Here is the thought process.  I or you come back here like 6-12 months later (i always use this example)...are you going to remember how this all works?  Maybe something change about this then the user of this would be like...what was this for?  If you point to like a commonly used thing like RDS which is well documented...then the person/you can understand.

Reply:
@sekka1  I've added the next documentation: 
https://github.com/ManagedKube/helm-charts/pull/55/files#diff-48733b875c465cb6d785c9e3b873be325a3c1c47094885a877bac85d74ca74b9R116

Reply:
I would also put that bit in here with the RDS flag (can't really remember which one..see its only been like 2 days) the ssl enforcement to true or something.  If that is set to true..you would need to use this section.

The reason to document is to help your future self or someone else coming along here so that they dont have to struggle through this like you did the last 4 days.  Just reading this...im not sure if it helps me to understand this and the problem fully...i suppose it helps.

Reply:
yeah you right
I posted that in the db connection string here 
https://github.com/ManagedKube/helm-charts/pull/55/files#diff-48733b875c465cb6d785c9e3b873be325a3c1c47094885a877bac85d74ca74b9R64

```
For db_connections ssl string use this template
DB_CONNECTION_STRING : mysql+mysqldb://user:password@server.com/db?ssl_ca=/etc/ssl/certs/query-exporter-cert-db.pem
```